code-scanning/frogbot-scan-pr.yml

```diff
@@ -0,0 +1,112 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
```

Comment:
Kindly add more details around the action used.

Reply:
Done :)